tests/factory_fixtures/lease.py

```diff
@@ -21,11 +19,28 @@ def _lease_attributes(unitNum, tenant, property):
 
 @pytest.fixture
 def create_lease(faker, lease_attributes, create_property, create_tenant):
-    def _create_lease(unitNum=None):
+    def _create_lease(
+        tenantId=None,
+        unitNum=None,
+        dateTimeStart=None,
+        dateTimeEnd=None,
+        propertyId=None,
+    ):
         if not unitNum:
             unitNum = faker.building_number()
-        tenant = create_tenant()
-        lease = LeaseModel(**lease_attributes(unitNum, tenant, create_property()))
+        if not dateTimeStart:
+            dateTimeStart = faker.date_time_this_decade()
+        if not dateTimeEnd:
+            dateTimeEnd = faker.date_time_this_decade(before_now=False, after_now=True)
+        if not tenantId:
+            tenantId = create_tenant().id
+        if not propertyId:
+            propertyId = create_property().id
+        lease = LeaseModel(
+            **lease_attributes(
+                unitNum, tenantId, propertyId, dateTimeStart, dateTimeEnd
+            )
+        )
```

Comment:
I actually had to go in and alter these fixtures in my own PR last week, and it looks like we had similar ideas 😄 

Here's how I did it in that PR:
https://github.com/codeforpdx/dwellinglybackend/blob/0dbcf73acf7fdd40475e468259e45d61928e3114/tests/factory_fixtures/lease.py#L6-L32

Basically, the main difference is that in mine I passed in the actual tenant and property rather than the ids, and used short-circuit evaluation in `_create_lease()` to make it more concise.  I think it looks a little better without all the `if not` blocks, but if you think they make the code more understandable, I'm open to hearing the reasoning.

I wouldn't bring this up, except I think this will probably result in some type of merge conflict, so it's probably best if we match them.  

Would you be open to using my style, with your added datetime parameters?

Reply:
Yes, it's no problem with me! The only reason I used ID was because the ID was currently the only field used, so I figured I'd just simplify things. I will have to confirm that this will also work, but I'm pretty sure it will.

Reply:
Done, using your code now. Still changed the fixtures a bit.